appveyor.yml

```diff
@@ -0,0 +1,12 @@
+version: '{branch}-1.0.{build}'
```

Comment:
I don't think we really care about that number, we just check if it builds, we don't use the artifacts.